doc/manual/io.rst

```diff
@@ -339,6 +339,9 @@ annotation  data can be converted to a trigger channel (STI 014) using an
 annotation map file which associates an annotation label with a number on
 the trigger channel.
 
+Saving EDF files is not supported natively (mne 0.18) yet. 
+This (gist)[https://gist.github.com/skjerns/bc660ef59dca0dbd53f00ed38c42f6be] can be used to save any mne.io.Raw into EDF/EDF+/BDF/BDF+.
```

Comment:
this will not render fine as you wrote a link as in .md and we use .rst for sphinx.

Reply:
my bad. priming of using the github online editor :) I changed and it should be fine now, I hope.